Resources/public/less/layouts.less

```diff
@@ -526,527 +536,527 @@ aside {
-                    color: @linkColorHover;
-                    &:before {
-                        color: @orange;
-                        text-shadow: none;
-                    }
-                }
-            }
+      }
+      .tag-amount {
+        display: block;
+        float: left;
+        color: @grayLow;
+        border-left: 1px solid @grayLight;
+        padding: 2px 5px;
+      }
+      &:hover,
+      &:focus {
+        text-decoration: none;
+        .tag-title {
+          color: @linkColorHover;
+          &:before {
+            color: @orange;
+            text-shadow: none;
+          }
         }
+      }
     }
+  }
 }
 
 .content-view-children {
-    .attribute-header {
-        h1, h2, h3, h4, h5, h6 {
-            border-bottom: 1px solid @grayLighter;
-            margin-bottom: 5px;
-        }
+  .attribute-header {
+    h1, h2, h3, h4, h5, h6 {
+      border-bottom: 1px solid @grayLighter;
+      margin-bottom: 5px;
     }
-    .content-view-line {
-        margin-bottom: 1em;
-        .clearfix();
-        .with-product-number {
-            padding-right: 150px;
-            position: relative;
-            display: block;
-            .product-number {
-                display: block;
-                position: absolute;
-                right: 0;
-                bottom: 0;
-                width: 150px;
-                font-size: 13px;
-                border-bottom: 1px solid @grayLighter;
-                color: @grayLow;
-                text-align: right;
-            }
-        }
-        form {
-            .item-price,
-            .item-buying-action {
-                text-align: right;
+  }
+  .content-view-line {
+    margin-bottom: 1em;
+    .clearfix();
+    .with-product-number {
+      padding-right: 150px;
+      position: relative;
+      display: block;
+      .product-number {
+        display: block;
+        position: absolute;
+        right: 0;
+        bottom: 0;
+        width: 150px;
+        font-size: 13px;
+        border-bottom: 1px solid @grayLighter;
+        color: @grayLow;
+        text-align: right;
+      }
+    }
+    form {
+      .item-price,
+      .item-buying-action {
+        text-align: right;
+      }
+      .item-price {
+        margin-bottom: 13px;
+      }
+    }
+  }
+  .contacts-container {
+    article {
+      margin-top: 10px;
+      .person-position {
+        margin-bottom: 0;
+      }
+      .image-and-data-set {
+        margin-top: 10px;
+        padding-bottom: 10px;
+        ul {
+          margin: 0;
+          padding: 0;
+          li {
+            &:before {
+              content: "";
+              height: 19px;
+              width: 19px;
+              margin-right: 0.5em;
+              display: inline-block;
+              #gradient > .vertical(@gray, @grayLow);
+              .border-radius(3px);
+              vertical-align: text-bottom;
+              position: relative;
+              left: auto;
+              top: auto;
```

Comment:
Was that element absolutely positioned earlier? In fot, please remove `top`, `left` properties.